.gitignore

```diff
@@ -38,15 +21,32 @@ $RECYCLE.BIN/
 .TemporaryItems
 .Trashes
 .VolumeIcon.icns
+.com.apple.timemachine.donotpresent
 
 # Directories potentially created on remote AFP share
 .AppleDB
 .AppleDesktop
 Network Trash Folder
 Temporary Items
 .apdisk
-nbproject
-src/java/version.properties
```

Comment:
Yes, you will want to keep version.properties in this gitignore.  I tried to give a live time demo, but had trouble because the app wouldn't start.  I pulled down your code, and my properties file was overwritten, which broke my environment.  Big no-no.

Reply:
I apologize for not including it in my gitignore.